ts/Data/DataTable.ts

```diff
@@ -851,7 +875,15 @@ class DataTable extends DataTableCore implements DataEvent.Emitter {
         const column = table.columns[columnName];
 
         if (column) {
-            const rowIndex = column.indexOf(cellValue, rowIndexOffset);
+            let rowIndex = -1;
+
+            if (Array.isArray(column)) {
+                // Normal array
+                rowIndex = column.indexOf(cellValue, rowIndexOffset);
+            } else if (defined(cellValue) && Number.isFinite(cellValue)) {
```

Comment:
This condition as far as I can see can be replaced with `if (isNumber(cellValue)) {`, where `isNumber` is from Utilities.ts. When I do that, there is also no need for typecasting the `cellValue` with +.